src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -30,83 +30,84 @@ limitations under the License.
 
     <command source="client" code="0x00" name="LaunchContent" response="LaunchResponse" optional="true">
       <description>Upon receipt, this SHALL launch the specified content with optional search criteria.</description>
-      <arg name="search" type="ContentSearch"/>
-      <arg name="autoPlay" type="BOOLEAN"/>
-      <arg name="data" type="CHAR_STRING" optional="true"/>
+      <arg name="Search" type="ContentSearchStruct"/>
+      <arg name="AutoPlay" type="BOOLEAN"/>
+      <arg name="Data" type="CHAR_STRING" optional="true"/>
     </command>
 
     <command source="client" code="0x01" name="LaunchURL" response="LaunchResponse" optional="true">
       <description>Upon receipt, this SHALL launch content from the specified URL.</description>
-      <arg name="contentURL" type="CHAR_STRING"/>
-      <arg name="displayString" type="CHAR_STRING" optional="true"/>
-      <arg name="brandingInformation" type="BrandingInformation" optional="true"/>
+      <arg name="ContentURL" type="CHAR_STRING"/>
+      <arg name="DisplayString" type="CHAR_STRING" optional="true"/>
+      <arg name="BrandingInformation" type="BrandingInformationStruct" optional="true"/>
     </command>
 
     <command source="server" code="0x02" name="LaunchResponse" optional="true" disableDefaultResponse="true">
       <description>This command SHALL be generated in response to LaunchContent command.</description>
-      <arg name="status" type="ContentLaunchStatusEnum"/>
-      <arg name="data" type="CHAR_STRING" optional="true"/>
+      <arg name="Status" type="ContentLaunchStatusEnum"/>
+      <arg name="Data" type="CHAR_STRING" optional="true"/>
     </command>
 
   </cluster>
 
-  <struct name="ContentSearch">
+  <struct name="ContentSearchStruct">
     <cluster code="0x050a"/>
-    <item name="parameterList" type="Parameter" array="true" optional="false"/>
+    <item name="ParameterList" type="ParameterStruct" array="true" optional="false"/>
   </struct>
 
-  <struct name="AdditionalInfo">
+  <struct name="AdditionalInfoStruct">
     <cluster code="0x050a"/>
-    <item name="name" type="CHAR_STRING"/>
-    <item name="value" type="CHAR_STRING"/>
+    <item name="Name" type="CHAR_STRING"/>
+    <item name="Value" type="CHAR_STRING"/>
   </struct>
 
   <enum name="MetricTypeEnum" type="ENUM8">
     <cluster code="0x050a"/>
-    <item name="PIXELS" value="0x00"/>
-    <item name="PERCENTAGE" value="0x01"/>
+    <item name="Pixels" value="0x00"/>
+    <item name="Percentage" value="0x01"/>
   </enum>
 
-  <struct name="Dimension">
+  <struct name="DimensionStruct">
     <cluster code="0x050a"/>
-    <item name="width" type="double" optional="false"/>
-    <item name="height" type="double" optional="false"/>
-    <item name="metric" type="MetricTypeEnum"/>
+    <item name="Width" type="double" optional="false"/>
+    <item name="Height" type="double" optional="false"/>
+    <item name="Metric" type="MetricTypeEnum"/>
```

Comment:
should we set optional=false here as well? or is that the default for structs?

Reply:
That's the default.